assets/src/components/amp-notice/style.css

```
@@ -28,25 +28,41 @@
 	background-color: #effbff;
 }
 
+.amp-notice--info svg {
+	color: #285be7;
+}
+
 .amp-notice--warning {
+	background-color: #fff9c8;
+}
+
+.amp-notice--warning svg {
+	color: #ff9f00;
```

I don't think the "warning" and "information" icons should be the same. I would recommend using the exclamation symbol (!) for the warning icon. Since "i" is already used for the information icon, I suppose it could simply be flipped vertically.

```suggestion
	color: #ff9f00;
	transform: rotate(180deg);
```

---

This works well.